### ISSUE TITLE
Where are the Factor communities and enthusiasts?

### ISSUE THREAD

**Issue by kytrinyx** [exercism/factor#49]
As we move towards the launch of the new version of Exercism we are going to be ramping up on actively recruiting people to help provide feedback.

Our goal is to get to 100%: everyone who submits a solution and wants feedback should get feedback. Good feedback. You can read more about this aspect of the new site here: http://mentoring.exercism.io/

To do this, we're going to need a lot more information about where we can find language enthusiasts.

* Is Factor supported by one or more large organizations?
* Does Factor have an official community manager?
* Do you know of specific communities (online or offline) that are enthusiastic about Factor? (Chat communities, forums, meetups, student clubs, etc)
* Are there popular conferences for Factor? (If so, what are some examples?)
* Are there any organizations who are targeted specifically at getting certain subgroups or demographics interested in Factor? (e.g. kids, teenagers, career changers, people belonging to various groups that are typically underrepresented in tech?)
* Are there specific groups or programs dedicated to mentoring people in Factor?
* Are there popular newsletters for Factor?
* Is Factor taught at programming bootcamps? (If so, what are some examples?)
* Is Factor taught at universities? (If so, what are some examples?)

In other words: where do people care a lot and/or know a lot about Factor?

This is part of the project being tracked in https://github.com/exercism/meta/issues/103

**Comment by hiljusti** [comment]
Communities:

- [Reddit](https://www.reddit.com/r/factor/)
- [Slack](https://slofile.com/slack/factorcode)
- [Gitter](https://gitter.im/factorcode/Lobby)
- Also Mailing list and blogs found on [the main website](https://factorcode.org/)

For other venues, I think Factor is more of an "underground" language. There aren't really conferences or organizations.

In my experience, people interested in Factor fall into 3 rough categories:

- Academics interested in unique properties of concatenative languages
- Curious devs interested in esoteric languages (e.g. exploring on sites like codewars.com)
- Senior embedded engineers who made careers off of Forth instead of C

I'm not sure of any companies that have notable success stories with Factor yet, but there are certainly [a few academic publications](https://concatenative.org/wiki/view/Concatenative%20language/Publications).

**Comment by hiljusti** [comment]
Since the tracking project has closed, I'll also close this.
